packages/clerk-js/src/v4/UserProfile/MfaPage.tsx

```diff
@@ -1,129 +1,85 @@
-import { PhoneNumberResource } from '@clerk/types';
+import { VerificationStrategy } from '@clerk/types';
 import React from 'react';
 
-import { useCoreUser } from '../../ui/contexts';
-import { useWizard, Wizard } from '../common';
-import { Col, Text } from '../customizables';
-import { ArrowBlockButton, useCardState, withCardStateProvider } from '../elements';
-import { getFlagEmojiFromCountryIso, handleError, parsePhoneString, stringToFormattedPhoneString } from '../utils';
+import { useCoreUser, useEnvironment } from '../../ui/contexts';
+import { Col, Grid, Text } from '../customizables';
+import { useCardState, withCardStateProvider } from '../elements';
+import { AuthenticatorApp, Mobile } from '../icons';
 import { FormButtonContainer } from './FormButtons';
+import { MfaPhoneCodePage } from './MfaPhoneCodePage';
+import { MfaTOTPPage } from './MfaTOTPPage';
 import { NavigateToFlowStartButton } from './NavigateToFlowStartButton';
 import { ContentPage } from './Page';
-import { AddPhone, VerifyPhone } from './PhonePage';
-import { SuccessPage } from './SuccessPage';
-import { AddBlockButton } from './UserProfileBlockButtons';
+import { TileButton } from './TileButton';
+import { getSecondFactors } from './utils';
 
 export const MfaPage = withCardStateProvider(() => {
-  const title = 'Add authentication step';
+  const card = useCardState();
+  const { attributes } = useEnvironment().userSettings;
```

Comment:
🙃 
Super minor but when we choose desctructuring, we do many levels in a single destructuring statement e.g.:
`const { userSettings: { attributes } } = useEnvironment();`

Reply:
Copy pasta, fixing